### ISSUE TITLE
v20200102 mobile client keepalive

### ISSUE THREAD

**Issue by dwsr36** [txthinking/brook#633]
### Prerequisites

* [x] Did you check the [wiki](https://github.com/txthinking/brook/wiki)?
* [x] Did you search in [issues](https://github.com/txthinking/brook/issues)?
* [x] Are you running the latest version?
* [x] I will write this according to this template.
* [x] I will write this in English.

### Environment

* Brook Version: v20200102
* Server Operating System: CentOS
* Server Operating System Version: 7
* Client Operating System: iOS
* Client Operating System Version: 12

### Expected Behavior

Connections can keep alive in mobile client websocket mode.

### Current Behavior

TCP keepalive timeout is seemly very short (approximately 10 seconds) on iOS client. It may lead to stability problems since connections are reestablished frequently, especially in a poor network environment.

But connections can keep for several minutes when I visit the same website page (e.g. https://www.google.com/) using Brook CLI on my computer.

### Steps to Reproduce

`brook wsserver -l :[port] -p [password] --tcpTimeout 500`

No matter how I change the timeout value, the connection time appears to be very short through iOS client.

**Comment by dwsr36** [comment]
I've found that when I was using iOS client, connections were closed and opened frequently, then the connections may be interrupted (it depends on the network I am using). Or it is just your design? @txthinking 

But I'd like to control the lasting time of connection using options from the server.

**Comment by txthinking** [comment]
The built-in parameters of GUI Client is:
```
--tcpTimeout 60
--tcpDeadline 0
--udpDeadline 60 (it is actually UDP over TCP)
```
same as CLI default

Looks like I should consider customizing the parameters in GUI Client

**Comment by dwsr36** [comment]
It seems that it is not caused by default parameters, TCP connections could last for minutes in iOS client v20190601 (using Brook mode and `--tcpDeadline 500` on the server).

I've also tested on my computer using the latest Brook CLI. My configuration is
```
brook wsclient -l 127.0.0.1:1080 -i 127.0.0.1 -s ws://[server_address]:[port] -p [password]
```
I did not specify other parameters but the connections can keep alive as before.

BTW, WebSocket mode is quite helpful, as original Brook mode has been invalid in some network environments.
